task3.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Apple pie recipe</title>
+    </head>
+    <body>
+        <h1>Applie pie</h1>
+        <br>
+        <img src="Imagess/apple-pie.jpg" alt="Applie-piee">
+        
+
+        <h2><ins>Total Time</ins></h2>

```

Comment:
Pretpostavljam da je `<ins>` element korišten zbog vizualnog efekta podvučenog teksta. Semantički njegova upotreba ovdje nema smisla jer se ne događa ubacivanje teksta u zamjenu za neki uklonjeni tekst.

Vizualno, za to služi element `<u>`. Međutim, upotreba "isključivo vizualnih" elemenata ne bi trebala postati praksa zbog toga što se vizuali uređuju putem CSS-a i ne bi trebali biti vezani uz HTML.

```suggestion
        <h2>Total Time</h2>
```